### PR TITLE
Magento_ProductAlert: avoid using deprecated escape* methods from Abs…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Static Tests

File: app/code/Magento/ProductAlert/view/frontend/templates/email/email.phtml [hunk 1]
```diff
@@ -4,14 +4,17 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block Magento\Framework\View\Element\Template */
+/**
+ * @var $block Magento\Framework\View\Element\Template
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/ProductAlert/view/frontend/templates/email/email.phtml]:
```suggestion
 * @var Magento\Framework\View\Element\Template $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/ProductAlert/view/frontend/templates/email/price.phtml [hunk 2]
```diff
@@ -4,23 +4,26 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\ProductAlert\Block\Email\Price */
+/**
+ * @var $block \Magento\ProductAlert\Block\Email\Price
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/ProductAlert/view/frontend/templates/email/price.phtml]:
```suggestion
 * @var \Magento\ProductAlert\Block\Email\Price $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/ProductAlert/view/frontend/templates/email/stock.phtml [hunk 3]
```diff
@@ -4,22 +4,25 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\ProductAlert\Block\Email\Stock */
+/**
+ * @var $block \Magento\ProductAlert\Block\Email\Stock
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/ProductAlert/view/frontend/templates/email/stock.phtml]:
```suggestion
 * @var \Magento\ProductAlert\Block\Email\Stock $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml [hunk 4]
```diff
@@ -4,10 +4,15 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /* @var $block \Magento\ProductAlert\Block\Product\View */?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml]: Please remove not needed opening and closing PHP tags

File: app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml [hunk 5]
```diff
@@ -4,10 +4,15 @@
...
+/**
+ * @var $block \Magento\ProductAlert\Block\Product\View
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml]:
```suggestion
 * @var \Magento\ProductAlert\Block\Product\View $block
 * @var \Magento\Framework\Escaper $escaper
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/ProductAlert/view/frontend/templates/email/price.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/ProductAlert/view/frontend/templates/email/price.phtml]: fixed

File: app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml]: fixed

File: app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/ProductAlert/view/frontend/templates/product/view.phtml]: fixed

File: app/code/Magento/ProductAlert/view/frontend/templates/email/stock.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/ProductAlert/view/frontend/templates/email/stock.phtml]: fixed

File: app/code/Magento/ProductAlert/view/frontend/templates/email/email.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/ProductAlert/view/frontend/templates/email/email.phtml]: fixed

**Comment by sergeynezbritskiy** [comment]
@magento run Static Tests

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8777](https://jira.corp.magento.com/browse/ENGCOM-8777) has been created to process this Pull Request

**Comment by ihor-sviziev** [comment]
@magento run Database Compare

**Pull request closed without merging**